tests/unit/services/activity-watcher-test.ts

```diff
@@ -0,0 +1,171 @@
+          timestamp: 123458,
+          data: {
+            influencer_name: 'bozito',
+            influencer_avatar: 'https://avatar.com/is-a-bad-movie',
+            campaign_name: 'bozocampaign',
+            url: 'https://entity.url.com'
+          }
+        },
+        wantInfoTitle: `<img class=\"toast-title__avatar\" src=\"https://avatar.com/is-a-bad-movie\" />
+     <i class=\"toast-title__icon upf-icon upf-icon--messages\"></i>`,
+        wantInfoMessage: `Draft by <b>bozito</b> in <b>bozocampaign</b>
+           <a href=\"https://entity.url.com\" target=\"_blank\">Review</a>`,
+      },
+      {
+        notification: {
+          notification_type: 'list_recommendation',
+          timestamp: 123458,
+          data: {
+            count: 20,
+            list_name: 'bozolist',
+            url: 'https://entity.url.com'
+          }
+        },
+        wantInfoTitle: "<i class=\"toast-title__icon upf-icon upf-icon--messages\"></i>",
+        wantInfoMessage: `You have <b>20</b> new recommendations for your <b>bozolist</b> list
+           <a href=\"https://entity.url.com\" target=\"_blank\">View</a>`,
+      },
+      {
+        notification: {
+          notification_type: 'thread_failure_summary',
+          timestamp: 123458,
+          data: {
+            mailing_url: 'https://entity.url.com'
+          }
+        },
+        wantErrorTitle: "<i class=\"fa fa-info-circle\" aria-hidden=\"true\"></i>",
+        wantErrorMessage: `<b>Mailing error.</b> We ran into a problem with one of your Mailings.
+          <a href="https://entity.url.com" target="_blank"><b>View my mailing</b></a>`
+      },
+      {
+        notification: {
+          notification_type: 'credential_disconnected',
+          timestamp: 123458,
+          data: {
+            integration_name: 'woop',
+            integration_url: 'https://inte-gration.com'
+          }
+        },
+        wantErrorTitle: "<i class=\"fa fa-info-circle\" aria-hidden=\"true\"></i>",
+        wantErrorMessage: `<b>Your woop has been disconnected.</b> Please check your integration.
+          settings and reconnect it to avoid any issues. <a href="https://inte-gration.com" target="_blank"><b>Reconnect</b></a>`
+      },
+    ];
+
+    eventTypesTestCases.forEach((testCase: any) => {
+      test('it dispatches events', function(assert) {
+        sinon.stub(this.owner.lookup('service:toast'), 'info').callsFake((message: string, title: string) => {
+          assert.equal(message, testCase.wantInfoMessage);
+          assert.equal(title, testCase.wantInfoTitle);
+        });
+        sinon.stub(this.owner.lookup('service:toast'), 'error').callsFake((message: string, title: string) => {
+          assert.equal(message, testCase.wantErrorMessage);
+          assert.equal(title, testCase.wantErrorTitle);
+        });
+
+        this.eventService.dispatch({
+          resource: '/notification/some-uu-id',
+          payload: testCase.notification,
+        });
```

Comment:
```suggestion
        const toastInfoStub = sinon.stub(this.owner.lookup('service:toast'), 'info').callsFake((message: string, title: string) => {
          assert.equal(message, testCase.wantInfoMessage);
          assert.equal(title, testCase.wantInfoTitle);
        });
        const toastErrorStub = sinon.stub(this.owner.lookup('service:toast'), 'error').callsFake((message: string, title: string) => {
          assert.equal(message, testCase.wantErrorMessage);
          assert.equal(title, testCase.wantErrorTitle);
        });

        this.eventService.dispatch({
          resource: '/notification/some-uu-id',
          payload: testCase.notification,
        });
        
        assert.true(toastInfoStub.calledOnce)
        assert.true(toastErrorStub.calledOnce)
```
To be sur that the toast service is call, ckeck if stub is called

Reply:
> To be sur that the toast service is call, ckeck if stub is called

That's not true in all cases, but I see your point.

Reply:
Maybe add some like:
```typescript
if(testCase.wantErrorTitle) {
  assert.true(toastErrorStub.calledOnce)
}
```
And same thing with toast info, what do you think?

Reply:
Yep, I had the same thing in mind :+1: